src/components/ExpandedViewMode/ExpandedViewModeContent/ActivityHistory/ActivityHistory.js

```diff
@@ -1,5 +1,51 @@
-import React from 'react'
+import React, { Component } from 'react'
+import './ActivityHistory.css'
+import { fetchGoalHistory } from '../../../../goal-history/actions'
+import { connect } from 'react-redux'
 
-export default function ActivityHistory() {
-  return <h1>Historial</h1>
+class ActivityHistory extends Component {
+  render() {
+    this.props.fetchGoalHistory({
```

Comment:
You can't do an async call, like `fetchGoalHistory` within the render function of a React class component. Should be in the lifecycle callback, such as
`componentDidMount`
https://reactjs.org/docs/state-and-lifecycle.html#adding-lifecycle-methods-to-a-class

Reply:
This comment needs to be resolved before merge